staging/src/k8s.io/apimachinery/pkg/runtime/serializer/cbor/internal/modes/decode.go

```diff
@@ -100,6 +104,12 @@ var Decode cbor.DecMode = func() cbor.DecMode {
 		// Reject the arbitrary-precision integer tags because they can't be faithfully
 		// roundtripped through the allowable Unstructured types.
 		BignumTag: cbor.BignumTagForbidden,
+
+		// When unmarshaling a byte string into a []byte, assume that the byte string
+		// contains base64-encoded bytes, unless explicitly counterindicated by an "expected
+		// later encoding" tag. This is consistent with the because of unmarshaling a JSON
```

Comment:
> When unmarshaling a byte string into a []byte, assume that the byte string contains base64-encoded bytes

I'm a bit confused... how would we get such a CBOR byte-string containing base64-encoded?

If the cbor content was obtained by converting from unstructured json / yaml → cbor (like an untyped client would do), the cbor encoding would be a utf-8 string (tag 35?) which *happened* to contain base64 data, right?

If the cbor content was obtained by *encoding* a typed struct with a []byte field, wouldn't the natural encoding just be an explicit cbor byte string (tag 48) without base-64 encoding?